CMake/lrs_options.cmake

```diff
@@ -8,6 +8,7 @@ option(BUILD_WITH_TM2 "Build with support for Intel TM2 tracking device" ON)
 option(BUILD_EASYLOGGINGPP "Build EasyLogging++ as a part of the build" ON)
 option(BUILD_WITH_STATIC_CRT "Build with static link CRT" ON)
 option(HWM_OVER_XU "Send HWM commands over UVC XU control" ON)
+option(COM_MULTITHREADED "call CoInitializeEx with COINIT_MULTITHREADED or COINIT_APARTMENTTHREADED flag" ON)
```

Comment:
Maybe "Set OFF to initialize COM library with COINIT_APARTMENTTHREADED  (Windows only)"